### DISCUSSION TITLE
Subscriptions do not update

### DISCUSSION THREAD

**Discussion by Jaldagar** [iBicha/playlet#408] [Q&A] [answered]
Why don't my subscriptions I have added update? Every time I go it's the same old videos. Any help appreciated!

**Comment by iBicha** [comment]
Do you see the subscriptions updated on your Invidious instance? Or only Playlet is not updated?

**Reply by Jaldagar** [reply]
Playlet was not updating, but I'm now seeing it could have potentially been that I didn't have it set to show most recent. Second issue is that I just used google takeout to export subscriptions for me to import into invidious and I am getting errors. I used the original CSV file and converted to JSON. Neither works. Ideally, I'd like to have that subscription data within invidious.

**Reply by iBicha** [reply, marked as answer]
Make sure you sort the subscriptions in the Invidious settings of your profile
<img width="617" alt="settings" src="https://github.com/iBicha/playlet/assets/17722782/008b82a6-a0b8-4d3d-9d35-7f6b766f50db">

**Reply by Jaldagar** [reply]
Thank you and somehow the csv/json data I imported is now working?!

**Reply by iBicha** [reply]
Great, glad this solved your issue

**Reply by Jaldagar** [reply]
One final question regarding Playlet and subscriptions. Will it always remain one line and have to scroll right? I wish there was a way to be able to fill the screen so I can go down like how I can on Invidious via computer.

**Reply by iBicha** [reply]
There's a ticket for this https://github.com/iBicha/playlet/issues/194 but I don't know when I can tackle it

**Reply by Jaldagar** [reply]
Unfortunately new videos are still failing to load via both invidious & playlet. Within hours of research yesterday it appears this has been happening for a very long time. What would  current fix be?

**Reply by iBicha** [reply]
Please see this issue https://github.com/iv-org/invidious/issues/3646
It helps if you restart your instance regularly.
You can also check some of your config, such as the pubsub for example https://github.com/iv-org/invidious/blob/1ae14cc22468ce6e0eb794752b113604b1d5582d/config/config.example.yml#L409

**Comment by Jaldagar** [comment]
Playlet wasn’t updating. I’m now seeing it could have potentially been that I didn’t have it set to show most recent. If so I was hoping all channels would automatically show most recent as YouTube does. Am I missing something? On Jun 19, 2024, at 11:51 AM, Brahim Hadriche ***@***.***> wrote:﻿
Do you see the subscriptions updated on your Invidious instance? Or only Playlet is not updated?
